packages/repeater/src/lib/RepeaterFactory.ts

```diff
@@ -1,46 +1,42 @@
-import { Repeater, RepeaterId } from './Repeater';
-import { RequestRunner, RequestRunnerOptions } from '../request-runner';
-import { RepeatersManager } from '../api';
+import { Repeater } from './Repeater';
 import { RepeaterOptions } from './RepeaterOptions';
+import { RepeaterBridgesOptions } from './RepeaterBridgesOptions';
+import { RequestRunner, RequestRunnerOptions } from '../request-runner';
 import { Configuration } from '@sectester/core';
 import { v4 as uuidv4 } from 'uuid';
 import { DependencyContainer, injectable, Lifecycle } from 'tsyringe';
+import { hostname } from 'os';
 
 /**
  *  A factory that is able to create a dedicated instance of the repeater with a bus and other dependencies.
  */
 @injectable()
 export class RepeaterFactory {
   private readonly MAX_NAME_LENGTH = 80;
-  private readonly repeatersManager: RepeatersManager;
   private readonly runnerOptions: Readonly<RequestRunnerOptions>;
 
   constructor(private readonly configuration: Configuration) {
-    this.repeatersManager =
-      this.configuration.container.resolve(RepeatersManager);
     this.runnerOptions =
       this.configuration.container.resolve(RequestRunnerOptions);
   }
 
   public async createRepeater({
-    projectId,
-    description,
     disableRandomNameGeneration,
-    namePrefix = 'sectester',
     requestRunnerOptions,
-    requestRunners
+    requestRunners,
+    namePrefix = hostname()
   }: RepeaterOptions = {}): Promise<Repeater> {
     await this.configuration.loadCredentials();
 
-    const { repeaterId } = await this.repeatersManager.createRepeater({
-      description,
-      projectId,
-      name: this.generateName(namePrefix, disableRandomNameGeneration)
-    });
-
     const container = this.configuration.container.createChildContainer();
 
-    container.register(RepeaterId, { useValue: repeaterId });
+    const repeaterBridgesOptions: RepeaterBridgesOptions = {
+      domain: this.generateName(namePrefix, disableRandomNameGeneration)
```

Comment:
Having random name generation at the client side enabled (the default behaviour) lead to uuid doubling in final repeater name at UI e.g. `DOSTRIKOV-bed362e3-bd70-4fd6-b400-99db1170b-6e117491-036d-4e93-9189-f5c97726a545`.

Also it eliminates `No such repeater` issue.